### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by zamzterz**
* Make them all run on Travis.
* Fix failing tests.

### CODE REVIEW

File: ui/src/test/java/com/schibsted/account/ui/login/IdentificationPresenterTest.kt [hunk 1]
```diff
@@ -24,46 +25,65 @@ import com.schibsted.account.ui.ui.InputField
...
                 verify(view).setPresenter(presenter)
             }
         }
 
         "verify input" should {
-            val view: IdentificationContract.View = mock()
-            val presenter = IdentificationPresenter(view, mock(), mock())
-
             "do nothing is the view isn't active" {
-                whenever(view.isActive).thenReturn(false)
```
Comment (sa-mao) [inline ui/src/test/java/com/schibsted/account/ui/login/IdentificationPresenterTest.kt]: Why remove this?

Comment (zamzterz) [reply ui/src/test/java/com/schibsted/account/ui/login/IdentificationPresenterTest.kt]: Not removed but replaced by the `viewMock` function (which does the same thing).

File: core/src/test/java/android/provider/Settings.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package android.provider;
```
[the remark this answers is not in the document]
Comment (zamzterz) [reply core/src/test/java/android/provider/Settings.java]: This was existing structure, all test code (including Kotlin tests) in `core/src/test` are under `java`.

File: config/lint.xml [hunk 3]
```diff
@@ -4,4 +4,5 @@
     <issue id="IconDensities" severity="ignore" />
     <issue id="RtlEnabled" severity="warning" />
     <issue id="GradleDependency" severity="ignore" />
+    <issue id="OldTargetApi" severity="ignore" />
```
[the remark this answers is not in the document]
Comment (zamzterz) [reply config/lint.xml]: Decided to get all tests running first, upgrading we'll be [addressed separately](https://github.com/schibsted/account-sdk-android/issues/413).

**Pull request merged**